### PR TITLE
Add person, student, and teacher classes

### PR DESCRIPTION
**Pull request by BranBayou**
### In this project I did the following
- created superclass Person.
- created subclass Student.
- created subclass Teacher.
- each class is saved in a separate file.
- checked for linter errors.

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
+    super(age, parent_permission: parent_permission, name: name)
+    @classroom = classroom
```
Comment (AmaduKamara) [inline student.rb:8]: ### [ Optional ]
Hi @BranBayou, what if you could have left the super to be like this: `super(age, parent_permission, name)` without reassigning values to the properties?

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(age, specialization, parent_permission: true, name: 'Unknown')
+    super(age, parent_permission: parent_permission, name: name)
+    @specialization = specialization
```
Comment (AmaduKamara) [inline teacher.rb:8]: ### [ Optional ]
Hi @BranBayou, what if you could have left the super to be like this: `super(age, parent_permission, name)` without reassigning values to the properties?

**Review by AmaduKamara — APPROVED** [review]
# Status: APPROVED 🟢 🟢 🟢

Hi @BranBayou,

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights: 
✅ You used Gitflow correctly 👍🏽
✅ The Linter check passed 👍🏽
✅ Created classes in a separate .rb file 👍🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: teacher.rb (hunk 2 above)
Comment (BranBayou) [reply teacher.rb:8]: Hi @AmaduKamara,
Thank you for your constructive feedback. I will do that.

**Pull request merged**